src/main/java/com/internet/shop/dao/jdbc/ProductDaoJdbcImpl.java

```diff
@@ -0,0 +1,112 @@
+package com.internet.shop.dao.jdbc;
+
+import com.internet.shop.dao.ProductDao;
+import com.internet.shop.exceptions.DataProcessException;
+import com.internet.shop.lib.Dao;
+import com.internet.shop.model.Product;
+import com.internet.shop.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ProductDaoJdbcImpl implements ProductDao {
+    @Override
+    public Product create(Product product) {
+        String query = "INSERT INTO products (productName, price) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+
+            PreparedStatement statement = connection
+                    .prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
+            statement.setString(1, product.getName());
+            statement.setDouble(2, product.getPrice());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                product.setId(resultSet.getLong(1));
+            }
+            return product;
+        } catch (SQLException e) {
+            throw new DataProcessException("Can't create product!", e);
+        }
+    }
+
+    @Override
+    public Optional<Product> get(Long productId) {
+        String query = "SELECT * FROM products WHERE product_id = ? AND deleted != 1";
```

Comment:
```suggestion
        String query = "SELECT * FROM products WHERE product_id = ? AND deleted = FALSE";
```
this way query would be more readable 
same with other queries

Reply:
done